presto-docs/src/main/sphinx/release/release-0.258.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.258
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix issues in SQL functions where query might fail with compiler error when function implementation includes lambda variable that has the same name as column names or function input names.
```

Comment:
Fix a bug in SQL functions where a query could fail with a compiler error if the function has a lambda variable with the same name as a column or function input.